Sources/Widgets/SVG/SVGLandmarkRepresentation/index.js

```diff
@@ -87,6 +119,16 @@ export function extend(publicAPI, model, initialValues = {}) {
 
   macro.setGet(publicAPI, model, ['circleProps', 'textProps', 'name']);
 
+  model.fontProperties = null;
+  if (initialValues.fromLineWidget === true) {
+    // this allows for different offset values when there are multiple instances of this class
+    model.dx = model.textProps.dx;
+    model.dy = model.textProps.dy;
+    model.fromLineWidget = true;
```

Comment:
not needed, initialValues.fromLineWidget is already in model since `Object.assign(model, DEFAULT_VALUES,initialValues)`